### PR TITLE
CommonServerPython - add skip update to return error in get remote data

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
added the string `skip update` to the error message returned from `return_error` when running the `get-modified-remote-data` command

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
+        message += 'skip update'
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4664]: add a space (or a new line) before the word `skip`
in your test, the resulting message is `Test Errorskip update`

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (Itay4) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py]: done

**Review by yaakovi — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
+        message += ' skip update'
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4664]: I would format it differently so it is clear that 'skip update' is not from the error message.
```suggestion
        message = 'skip update. error: ' + message
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4663]: `error` can be omitted (and usually is). In that case we should use sys.exc_info() to get the last exception and check that is in not `NotImplementedError`

**Review by glicht — CHANGES_REQUESTED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 3 above)
Comment (Itay4) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py]: done

**Review by glicht — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 3 above)
Comment (glicht) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py]: excellent

**Pull request merged**